### ISSUE TITLE
AttributeError: 'ActionModule' object has no attribute '_handle_template' when using mode: text in net_put

### ISSUE THREAD

**Issue by nickzxcv** [ansible-collections/ansible.netcommon#153]
##### SUMMARY
With this task running in ansible 2.9.14 from Ubuntu package 2.9.14-1ppa~xenial:
```
---
- name: COPY THE BASE CONFIG TEMPLATE
  net_put:
    src: templates/asa5516-day0-config.jinja
    dest: flash:asa5516-day0-config
    mode: text
```
I get the following error:
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: AttributeError: 'ActionModule' object has no attribute '_handle_template'

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
ansible.netcommon.net_put

##### ANSIBLE VERSION

```
ansible 2.9.14
  config file = /home/nicks/.ansible.cfg
  configured module search path = [u'/home/nicks/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/dist-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.12 (default, Jul 21 2020, 15:19:50) [GCC 5.4.0 20160609]
```

##### CONFIGURATION

```
ANSIBLE_PIPELINING(/home/nicks/.ansible.cfg) = True
ANSIBLE_SSH_CONTROL_PATH(/home/nicks/.ansible.cfg) = /tmp/ansible-ssh-%%h-%%p-%%r
DEFAULT_HOST_LIST(/home/nicks/.ansible.cfg) = [u'/home/nicks/tools/ansible/hosts']
DEFAULT_ROLES_PATH(/home/nicks/.ansible.cfg) = [u'/home/nicks/.ansible/roles', u'/home/nicks/ansible-repo/roles']
INTERPRETER_PYTHON(/home/nicks/.ansible.cfg) = /usr/bin/python3
PERSISTENT_COMMAND_TIMEOUT(/home/nicks/.ansible.cfg) = 30
```

##### OS / ENVIRONMENT
Ubuntu 16.04

##### STEPS TO REPRODUCE
```
---
- name: COPY THE BASE CONFIG TEMPLATE
  net_put:
    src: templates/asa5516-day0-config.jinja
    dest: flash:asa5516-day0-config
    mode: text
```

##### EXPECTED RESULTS
The templated file should copy to the Cisco ASA device.

##### ACTUAL RESULTS
```
< TASK [COPY THE BASE CONFIG TEMPLATE] >
 --------------------------------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||

task path: /home/nicks/ansible-repo/playbooks/cisco_asa/tasks/base-config.yaml:2
<vpn1.iad4.prod.example.com> attempting to start connection
<vpn1.iad4.prod.example.com> using connection plugin ansible.netcommon.network_cli
<vpn1.iad4.prod.example.com> local domain socket does not exist, starting it
<vpn1.iad4.prod.example.com> control socket path is /home/nicks/.ansible/pc/8586ebf559
<vpn1.iad4.prod.example.com> local domain socket listeners started successfully
<vpn1.iad4.prod.example.com> loaded cliconf plugin ansible_collections.cisco.asa.plugins.cliconf.asa from path /home/nicks/.ansible/collections/ansible_collections/cisco/asa/plugins/cliconf/asa.py for network_os cisco.asa.asa
<vpn1.iad4.prod.example.com>
<vpn1.iad4.prod.example.com> local domain socket path is /home/nicks/.ansible/pc/8586ebf559
Getting network OS from inventory
The full traceback is:
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/ansible/executor/task_executor.py", line 147, in run
    res = self._execute()
  File "/usr/lib/python2.7/dist-packages/ansible/executor/task_executor.py", line 665, in _execute
    result = self._handler.run(task_vars=variables)
  File "/usr/lib/python2.7/dist-packages/ansible/plugins/action/net_put.py", line 74, in run
    self._handle_template(convert_data=False)
AttributeError: 'ActionModule' object has no attribute '_handle_template'
fatal: [vpn1.iad4.prod.example.com]: FAILED! => {
    "msg": "Unexpected failure during module execution.",
    "stdout": ""
}
```

**Comment by ganeshrn** [comment]
@nschmalenberger The issue is fixed in ansible-2.10 and netcommon collection. I have raised a backport PR on stable-2.9 branch https://github.com/ansible/ansible/pull/72165. After it is merged in the stable-2.9 branch the fix will be available in the subsequent minor release of 2.9

**Comment by Qalthos** [comment]
I believe this is fixed by #103, and this issue is just here looking for a backport to 2.9, yes?
